fml/message_loop_unittests.cc

```diff
@@ -315,3 +316,32 @@ TEST(MessageLoop, CanCreateConcurrentMessageLoop) {
   latch.Wait();
   ASSERT_GE(thread_ids.size(), 1u);
 }
+
+TEST(MessageLoop, TIME_SENSITIVE(WakeUpTimersAreSingletons)) {
+  auto loop_impl = fml::MessageLoopImpl::Create();
+
+  const auto t1 = fml::TimeDelta::FromMilliseconds(10);
+  const auto t2 = fml::TimeDelta::FromMilliseconds(20);
+
+  const auto begin = fml::TimePoint::Now();
+
+  // Register a task scheduled for 10ms in the future. This schedules a
+  // WakeUp call on the MessageLoopImpl with that fml::TimePoint
+  loop_impl->PostTask(
+      [&]() {
+        auto delta = fml::TimePoint::Now() - begin;
+        auto ms = delta.ToMillisecondsF();
+        ASSERT_GE(ms, 18);
```

Comment:
This has to be at least 20 right? If not, that would mean the timer fired prematurely.